### PR TITLE
Check return output of mom bash version

### PR DESCRIPTION
**Pull request by RKORavi**
#### Describe Bug or Feature
Check return output of mom bash version. 
If it fails to get mom bash version then it results in : IndexError: list index out of range

#### Describe Your Change
Check if command execution to get mom version fails then return True

#### Link to Design Doc
NA

#### Attach Test and Valgrind Logs/Output

[TestNonprintingCharacters.zip](https://github.com/openpbs/openpbs/files/6435094/TestNonprintingCharacters.zip)

### CODE REVIEW

**Review by prakashcv13 — APPROVED** [review]

**Review by npadole20 — APPROVED** [review]

File: test/fw/ptl/lib/ptl_mom.py [hunk 1]
```diff
@@ -696,7 +696,11 @@ def check_mom_bash_version(self):
         ret = self.du.run_cmd(self.hostname, cmd=cmd, sudo=True,
                               as_script=True)
         req_bash_version = "4.2.46"
-        mom_bash_version = ret['out'][0]
+        if len(ret['out']) > 0:
+            mom_bash_version = ret['out'][0]
+        else:
+            # If it fails to get bash version return True
```
Comment (bhroam) [inline test/fw/ptl/lib/ptl_mom.py:702]: Comments should explain why, not what the code is doing.  Why are you returning true if we can't get the bash version?

**Review by bhroam — COMMENTED** [review]
I'm still not sure I understand.  If you need the bash version to be a certain version, and you can't get the version, shouldn't you skip the test?  Or are you saying that if you can't figure it out, try anyway?  If that's the case, the comment should say that.

**Comment by RKORavi** [comment]
> I'm still not sure I understand. If you need the bash version to be a certain version, and you can't get the version, shouldn't you skip the test? Or are you saying that if you can't figure it out, try anyway? If that's the case, the comment should say that.

@bhroam please let me know if comment is clear now.

**Comment by bhroam** [comment]
@RKORavi I'm not sure the comment is helpful.  Let me know if this is wrong: If we can't get the bash version, there is no harm in trying to run the test.  It might fail in an error, but at least we tried, right?

Otherwise I'm still not understanding why you want to run the test if you can't find the bash version.  You are querying the bash version to see if it will run properly.  Why are you still running the test if you don't know if it will run properly?

**Review by prakashcv13 — APPROVED** [review]

**Pull request merged**